README.md

```diff
@@ -3,18 +3,47 @@ A list of resources for data science work on climate change
 
 ## Data Sets
 
+### Global
+* [ICE Embodied Carbon Database](https://circularecology.com/embodied-carbon-footprint-database.html)
+* [Carbon in food downloadable csv](https://ourworldindata.org/food-choice-vs-eating-local)
+* [EPW Weather files (for Ladybug on Grasshopper, but adaptable)](https://www.ladybug.tools/epwmap/)
+* [Renewable Ninja - front end for NASA dataset](https://www.renewables.ninja/)
+* [One of many NASA datasets. This one covers precipitation](https://disc.gsfc.nasa.gov/datasets/GPM_3IMERGHH_06/summary?keywords=IMERG)
```

Comment:
This and renewable ninja are front ends for more canonical data sets. Maybe we should have a new heading for GUI or similar?

Reply:
possibly. There's a definite role for 'easy to use' data set guis to help people get started.